graspologic/plot/__init__.py

```diff
@@ -4,6 +4,24 @@
 import sys
 import matplotlib as mpl
 
-from .plot import heatmap, gridplot, pairplot, degreeplot, edgeplot, screeplot
+from .plot import (
```

Comment:
@bdpedigo I am a little confused as to why I am getting "ERROR graspy/plot/__init__.py - ValueError: attempted relative import beyond ... " from the test-coverage check. The other files have similar calls to files in other folders and they do not seem to error.

Reply:
need more info about the specific error you are getting (and what line is throwing it). also post in discussion channel? i suspect some folks there will also be able to help